util/find-doc-nits

```diff
@@ -444,6 +444,21 @@ sub check {
         check_section_location($id, $contents, "EXAMPLES", "SEE ALSO");
     }
 
+    # Check for proper in-man-3 API links.
```

Comment:
This comment should probably be moved down to the next while.

Reply:
fixed.  added a comment before that while, tho.